src/rules/no-unsafe-values.js

```diff
@@ -13,6 +19,11 @@ export default {
 
 		messages: {
 			unsafeNumber: "Number outside safe range found.",
+			unsafeInteger: "Integer outside safe range found.",
+			unsafeZero:
+				"This number will evaluate to zero, which is unintended.",
```

Comment:
Can we include the number to make this a bit clearer?
```suggestion
				"The number '{{ value }}` will evaluate to zero.",
```

Reply:
There's nothing interesting in the number.  It's either going to be exactly the same as the highlighted text (if we use txt) or 0 (if we use node.value)

Reply:
That's true when people are using ESLint in an editor. People also use ESLint in their CI system, where it just outputs messages onto the terminal. In that case, having the value in the message is really helpful.